### PR TITLE
build: cacao patch v1.1.1 clockskew

### PR DESCRIPTION
**Pull request by zachferland**
- cacao patch v1.1.1 clockskew
- dids v3.2.0 with cacao phaseout
- abort controller lib change

### CODE REVIEW

**Comment by zachferland** [comment]
hmm not sure about ci build issue yet, do not see this locally

File: packages/key-did-resolver/src/nist_weierstrass_common.ts [hunk 1]
```diff
@@ -8,8 +8,8 @@ import { base64url } from 'multiformats/bases/base64'
   * For BigInt see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
   */
 interface BigIntPoint {
-   x: BigInt,
-   y : BigInt
+   x: bigint,
+   y : bigint
```
Comment (oed) [inline packages/key-did-resolver/src/nist_weierstrass_common.ts:12]: Why this change?

**Review by oed — COMMENTED** [review]
one question otherwise lgtm

**Comment by stbrody** [comment]
@zachferland can you explain why this change made the abort controller issue show up?  I don't see you using abort controller in your code anywhere, is it used in one of our dependencies?  If it's one of the dependencies we control, should we fix it there?

File: packages/key-did-resolver/src/nist_weierstrass_common.ts (hunk 1 above)
Comment (oed) [reply packages/key-did-resolver/src/nist_weierstrass_common.ts:12]: Why is your linter different from the rest of the teams?

Comment (oed) [reply packages/key-did-resolver/src/nist_weierstrass_common.ts:12]: I don't understand why this change is needed based on the changes you did in the code.

**Comment by zachferland** [comment]
@stbrody yeah its only due to upgrading dependencies (ref issue linked above, also seen in earlier CI fail), likely due to node types or typescript dependencies upgrading (not sure exactly unfortunately). But the current abort-controller library wont align on types now until they change it. 

even if we tried to only up dependencies needed here in package lock, would likely run into same issue soon 

we use abort-controller in common

**Comment by zachferland** [comment]
basically node type and abort-controller pollyfill/lib dont match types now

**Comment by stbrody** [comment]
I will defer to @ukstv on this review

**Comment by stbrody** [comment]
FYI I'd like to do a release tomorrow with fixes for gitcoin and ideally this would be included in it if we can get to consensus on the changes in time

**Comment by ukstv** [comment]
I am not sure what happened there, but apparently it was something weird, considering changes in abort-controller package and package-lock file. Maybe simpler https://github.com/ceramicnetwork/js-ceramic/pull/2237 could work so that @zachferland does not have to redo the work in this PR?

**Comment by ukstv** [comment]
I do not know, maybe different version of npm contributed to the confusion?

**Comment by ukstv** [comment]
Can we close it now @zachferland ?
